.circleci/config.yml

```diff
@@ -397,7 +410,7 @@ jobs:
       - run:
           name: Unzip module artifact
           command: |
-            unzip `ls /workspace/$PACKAGE_NAME.Linux-ubuntu18.04-x86_64.*.zip |grep -v rce` -d /workspace/
+            unzip `ls /workspace/$PACKAGE_NAME.Linux-ubuntu18.04-x86_64.*.zip | grep -v rce` -d /workspace/
```

Comment:
Hardcoded path means other paths in the future are more work, Can this be removed?

Reply:
This is benchmarking stuff, not really part of this PR. I just added a space there, purely OCD thing.